src/core/account/account-pixie.ts

```diff
@@ -160,9 +161,20 @@ const accountPixie: TamePixie<AccountProps> = combinePixies({
         update() {
           if (input.props.accountOutput?.accountApi == null) return
 
+          const { accountId } = input.props
+          const { sessionKey } = input.props.state.accounts[accountId]
+          const { stashTree } = getStashById(
+            toApiInput(input),
+            sessionKey.loginId
+          )
+          const interval = stashTree.wipChange != null ? 5000 : SYNC_INTERVAL
+
+          dataTask.setDelay(interval)
+          loginTask.setDelay(interval)
+
           // Start once the EdgeAccount API exists:
-          dataTask.start({ wait: SYNC_INTERVAL * (1 + Math.random()) })
-          loginTask.start({ wait: SYNC_INTERVAL * (1 + Math.random()) })
+          dataTask.start({ wait: interval * (1 + Math.random()) })
```

Comment:
Do not change the `dataTask` frequency. The problem is with the login server, so we just need to increase that task's frequency.